README.md

```diff
@@ -4,13 +4,14 @@
 **IMP-MARL** offers a platform for benchmarking the scalability of cooperative MARL methods in real-world engineering applications.
 
 In IMP-MARL, you can:
-- [Implement your own infrastructure management planning (IMP) environment or execute an available IMP environment](environments/).
-- [Train IMP policies through state-of-the-art MARL methods. The environments can be integrated with typical ecosystems via wrappers](imp_wrappers/).
-- [Compute expert-based heuristic policies](./heuristics/)
+- [Implement your own infrastructure management planning (IMP) environment or execute an available IMP environment](imp_marl/environments/).
+- [Train IMP policies through state-of-the-art MARL methods. The environments can be integrated with typical ecosystems via wrappers](imp_marl/imp_wrappers/).
+- [Compute expert-based heuristic policies](papers/neurips_23/heuristics/)

```

Comment:
We could define a template class and incorporate it into imp_marl (in the future)